pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -219,12 +219,15 @@ func (s *DiscoveryServer) updateClusterInc(push *model.PushContext, clusterName
 	// arbitrary destination rule's subset labels!
 	labels := push.SubsetToLabels(nil, subsetName, hostname)
 
-	portMap, f := push.ServicePort2Name[string(hostname)]
+	ports, f := push.ServicePort2Name[string(hostname)]
 	if !f {
 		return s.updateCluster(push, clusterName, edsCluster)
 	}
-	svcPort, f := portMap.GetByPort(port)
-	if !f {
+
+	// Check that there is a matching port
+	// We don't use the port though, as there could be multiple matches
```

Comment:
What's this comments mean, you donot change any logic here?

Reply:
I thought it was not clear why we look up the port but do not use it (which is what I changed), so I added this comment to explain we don't use it as there could he multiple matching ports